### DISCUSSION TITLE
leaflet in pure javascript grafana

### DISCUSSION THREAD

**Discussion by yosiasz** [Leaflet/Leaflet#9269] [General]
GReetings

I would like to use leaflet in grafana using a `dynamic text plugin` [https://volkovlabs.io/plugins/volkovlabs-dynamictext-panel/](url) that allows to reference external resources.
Then you are able to use pure javascript to create what you need.

The issue I am running into is I get the error L is not defined when following the https://leafletjs.com/examples/quick-start/

Is there anything I am missing here? Can this be done using pure js?

Thanks

**Comment by rs1659** [comment]
I am currently getting the same error in Grafana 11 on a normal Text panel in HTML mode, which is especially interesting since I got it working in Grafana 10.
